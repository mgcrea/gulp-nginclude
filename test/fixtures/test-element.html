<!doctype html>
<html>
<head>
  <meta charset="utf-8">
  <title>Title</title>
</head>
<body ontouchstart ng-controller="MainCtrl">
  <ng-include src="foo" class="mydiv"></ng-include>
  <ng-include src="'partial-1.html'" class="mydiv"></ng-include>
  <ng-include src="'partial-2.html'" class="mydiv"></ng-include>
</body>
</html>
