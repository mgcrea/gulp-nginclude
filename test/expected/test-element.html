<!doctype html>
<html>
<head>
  <meta charset="utf-8">
  <title>Title</title>
</head>
<body ontouchstart="" ng-controller="MainCtrl">
  <ng-include src="foo" class="mydiv"></ng-include>
  <ng-include class="mydiv">
<!-- ngInclude: 'partial-1.html' -->
<p>foo</p>
<!--/ngInclude: 'partial-1.html' -->
</ng-include>
  <ng-include class="mydiv">
<!-- ngInclude: 'partial-2.html' -->
<p>bar</p>
<!--/ngInclude: 'partial-2.html' -->
</ng-include>
</body>
</html>
